src/app/modules/workspace/workspace.controller.ts

```diff
@@ -0,0 +1,298 @@
+import { celebrate, Joi, Segments } from 'celebrate'
+import { AuthedSessionData } from 'express-session'
+import { StatusCodes } from 'http-status-codes'
+import { ErrorDto } from 'shared/types'
+import { WorkspaceDto } from 'shared/types/workspace'
+
+import {
+  WORKSPACE_MAX_TITLE_LENGTH,
+  WORKSPACE_MIN_TITLE_LENGTH,
+} from '../../../../shared/constants'
```

Comment:
should this be an absolute path instead? 'shared/types/workspace'

Reply:
I realised in the BE for some reasoning my build will fail if i use absolute path 😩 

Hence I think other files also use relative like `src/app/modules/submission/receiver/receiver.service.ts`